packages/cosmic-swingset/src/chain-main.js

```diff
@@ -288,6 +288,12 @@ export default async function main(progname, args, { env, homedir, agcc }) {
   // instance, and we update the 'portNums.storage' value each time toSwingSet is called
   async function launchAndInitializeSwingSet(bootMsg) {
     // this object is used to store the mailbox state.
+    // TODO: Generalize StorageAPI caching for both makeChainStorage and buildCrankBuffer?
+    // const sendToSwingset = msg => chainSend(portNums.storage, msg);
```

Comment:
do these comments contain proposed code additions, or are they reminders of where the values come from, or what? If they're proposals, we should probably put them in an issue where we can have more context about what they're intended to do.

Reply:
They were conversation starters for proposed changes, which seem worth at least discussion. I have removed them and will followup in a separate PR.